evm/interchainTokenFactory.js

```diff
@@ -143,6 +143,7 @@ async function processCommand(config, chain, options) {
 
             const tokenId = await interchainTokenFactory.interchainTokenId(wallet.address, deploymentSalt);
             printInfo('tokenId', tokenId);
+            printInfo('token address', await interchainTokenService.interchainTokenAddress(tokenId));
```

Comment:
```suggestion
            printInfo('Token address', await interchainTokenService.registeredTokenAddress(tokenId));
```
Use the validated queries by default